### ISSUE TITLE
Managing images and running containers using hiera?

### ISSUE THREAD

**Issue by ricoli** [garethr/garethr-docker#192]
Currently there is no puppet class for docker images or containers, so I wonder if there is any way to use hiera configuration to manage them? Afaik we can only use hiera with classes... Any thoughts?

**Comment by garethr** [comment]
I think this is what you're looking for for containers: https://github.com/garethr/garethr-docker/blob/master/manifests/run_instance.pp

I'd be happy to see a PR for something similar (even better a bunch of examples) for images too.

**Comment by ricoli** [comment]
Ah yes indeed, I missed that class... yeah could do the same for the images so they can be prefetched.

**Comment by ricoli** [comment]
Leaving a temporary solution here for the time being - adding this to a manifest:
```
$dockerImages = hiera('docker::images', {})
create_resources('docker::image', $dockerImages)
```

**Comment by garethr** [comment]
Fixed in #194
